src/Events/Category_Colors/Event_Category_Meta.php

```diff
@@ -0,0 +1,330 @@
+<?php
+/**
+ * Event_Category_Meta class for taxonomy meta.
+ * Handles metadata for terms within the `tribe_events_cat` taxonomy.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Category_Colors
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Events\Category_Colors;
+
+use InvalidArgumentException;
+use Tribe__Events__Main;
+use WP_Term;
+use WP_Error;
```

Comment:
Please sort the imports alphabetically. @Camwyn and I will both thank you 😅 

Applies to all files, but I'm just calling it out here.

```suggestion
use InvalidArgumentException;
use Tribe__Events__Main;
use WP_Error;
use WP_Term;
```